packages/storage-proxy/src/cli/arguments.ts

```diff
@@ -0,0 +1,18 @@
+import { toEthereumAddress } from "@streamr/utils";
+import { Argument } from 'commander';
+import { StorageNodeMetadata, } from "streamr-client";
```

Comment:
Are we using Prettier for opinionated format?
This comment should have been removed.

Reply:
Yes, what's wrong with these lines?

Reply:
Comma** after `StorageNodeMetadata` should be auto-formatted to be removed.
Just making sure Prettier is being used.